fendermint/eth/api/src/apis/eth.rs

```diff
@@ -738,6 +740,23 @@ pub async fn get_storage_at<C>(
 where
     C: Client + Sync + Send,
 {
+    let encode = |data: Option<uints::U256>| {
+        let mut bz = [0u8; 32];
+        if let Some(data) = data {
+            data.to_big_endian(&mut bz);
+        }
+        // The client library expects hex encoded string.
+        Ok(hex::encode(bz))
+    };
+
+    let height = data.query_height(block_id).await?;
+
+    // If not an EVM actor, return empty.
+    if data.get_actor_type(&address, height).await? != ActorType::Known("evm".to_string()) {
```

Comment:
I think it would be worth creating `const` instances of known types [such as these](https://github.com/filecoin-project/ref-fvm/blob/daefd6bcb4eefe3e8c29454d73a295c3fbe81bfc/shared/src/error/mod.rs#L50-L57), and then we should be able to reuse it as `ActorType::Evm` and not have to repeat this, always guessing if it's upper or lower case, etc.

Reply:
Fixed as you suggested. Thanks!